src/mscorlib/src/System/Runtime/Intrinsics/X86/Sse2.cs

```diff
@@ -285,6 +351,35 @@ public static class Sse2
         /// </summary>
         public static Vector128<double> ConvertToDouble(Vector128<float> value) => ConvertToDouble(value);
 
+        /// <summary>
+        /// int _mm_cvtsd_si32 (__m128d a)
+        /// </summary>
+        public static int ConvertToInt32Scalar(Vector128<double> value) => ConvertToInt32Scalar(value);
+        /// <summary>
+        /// __int64 _mm_cvtsd_si64 (__m128d a)
+        /// </summary>
+        public static long ConvertToInt64Scalar(Vector128<double> value) => ConvertToInt64Scalar(value);
+        /// <summary>
+        /// __m128 _mm_cvtsd_ss (__m128 a, __m128d b)
+        /// </summary>
+        public static Vector128<float> ConvertToSingleScalar(Vector128<float> a, Vector128<double> b) => ConvertToSingleScalar(a, b);
+        /// <summary>
+        /// double _mm_cvtsd_f64(__m128d a)
+        /// </summary>
+        public static double ConvertToDoubleScalar(Vector128<double> value) => ConvertToDoubleScalar(value);
+        /// <summary>
+        /// __m128d _mm_cvtsi32_sd (__m128d a, int b)
+        /// </summary>
+        public static Vector128<double> ConvertToDoubleScalar(Vector128<double> a, int b) => ConvertToDoubleScalar(a, b);
+        /// <summary>
+        /// __m128d _mm_cvtsi64_sd (__m128d a, int b)
+        /// </summary>
+        public static Vector128<double> ConvertToDoubleScalar(Vector128<double> a, long b) => ConvertToDoubleScalar(a, b);
+        /// <summary>
+        /// __m128d _mm_cvtss_sd (__m128d a, __m128 b)
+        /// </summary>
+        public static Vector128<double> ConvertToDoubleScalar(Vector128<double> a, Vector128<float> b) => ConvertToDoubleScalar(a, b);
```

Comment:
MOVD/MOVQ instructions are missing from here (and AVX/AVX2). I propose something like this:
``` C#
// __m128i _mm_cvtsi32_si128 (int a)
public static Vector128<int> CopyInt32(int value);
// int _mm_cvtsi128_si32 ( __m128i a)
public static int CopyInt32(Vector128<int> value);
// __m128i _mm_cvtsi64_si128(__int64)
public static Vector128<long> CopyInt64(long value);
// __int64 _mm_cvtsi128_si64(__m128i)
public static long CopyInt64(Vector128<long> value);
```

Reply:
Fixed. Went with the existing `ConvertTo` naming convention